src/metadatas.rs

```diff
@@ -17,6 +19,7 @@ pub struct Metadata {
     pub trips_with_bike_info_count: usize,
     pub trips_with_wheelchair_info_count: usize,
     pub networks: Vec<String>,
+    pub networks_start_end_dates: Option<HashMap<String, Option<HashMap<String, String>>>>,
```

Comment:
I think it would be easier to read with a struct instead of an Hashmap<string, string> there.

Maybe you can use the interval struct and add serde tag to it?

Reply:
Yes you're right, I didn't have the reflex to use a Struct. Let me know if I've done it the right way @antoine-de.
Thanks !